source/stylesheets/base/_layout.scss

```diff
@@ -1,10 +1,6 @@
-body {
-	font-family: $font-family-base;
-}
-
 ul {
 	list-style: none;
-	padding-inline-start: 0px;
+	padding-inline-start: 0em;
```

Comment:
Interesting! I've never used this before. In looking at its cross-browser compatibility, IE doesn't support it (big surprise there..). Consider just setting its `padding-left` to 0 take care of compatibility.